bindings/ergo-lib-wasm/src/ast.rs

```diff
@@ -83,6 +91,46 @@ impl Constant {
             .map_err(to_js)
     }
 
+    /// Create `Coll[Int]` from string array
+    #[allow(clippy::boxed_local)]
+    pub fn from_i32_str_array(arr: Box<[JsValue]>) -> Result<Constant, JsValue> {
```

Comment:
Why use string and not JS number here? In i64 equivalent, the string was used since 64 bits do not fit in JS number.

Reply:
Yes, Good point I rewrite the function.